lib/render/mayaToHydra/renderGlobals.cpp

```diff
@@ -292,15 +294,24 @@ void MtohInitializeRenderGlobals() {
         auto* rendererPlugin =
             HdRendererPluginRegistry::GetInstance().GetRendererPlugin(
                 rendererDesc.rendererName);
-        if (rendererPlugin == nullptr) { continue; }
+        if (!rendererPlugin)
+            return false;
         auto* renderDelegate = rendererPlugin->CreateRenderDelegate();
-        if (renderDelegate == nullptr) { continue; }
+        if (!renderDelegate)
+            return false;
+
         const auto rendererSettingDescriptors =
             renderDelegate->GetRenderSettingDescriptors();
         _rendererAttributes[rendererDesc.rendererName] =
             rendererSettingDescriptors;
         delete renderDelegate;
 
+        if (renderer) {
+            std::unique_ptr<MtohRenderOverride> override(new MtohRenderOverride(rendererDesc));
```

Comment:
Maybe find a better word than: ```override``` _(Whilst it's not a problem here, it is a bit jarring to see keywords used as identifiers)_.